conf/app.routes

```diff
@@ -3,9 +3,10 @@
 GET        /overview                                controllers.EventReportController.getOverview
 GET        /versions                                controllers.EventReportController.getVersions
 
-POST       /save                                    controllers.EventReportController.saveEvent
+GET        /event                                   controllers.EventReportController.getEvent
 POST       /compile                                 controllers.EventReportController.compileEvent
 
 POST       /submit-event-declaration-report         controllers.EventReportController.submitEventDeclarationReport
 
-GET        /event                                   controllers.EventReportController.getEvent
+GET        /cache/user-answers                      controllers.EventReportController.getUserAnswersFromCache
```

Comment:
I think we shouldn't expose our backend structure as /cache/..

would it be better if we say only /user-answers ? 

Method name can stay as is.

Reply:
I have renamed